csr_check.go

```diff
@@ -75,14 +76,9 @@ func validateCSRContents(req *certificatesv1beta1.CertificateSigningRequest, csr
 		return "", fmt.Errorf("Too few usages")
```

Comment:
This error message could really do with improvement couldn't it 🤔

Reply:
I'll work on this in https://github.com/openshift/cluster-machine-approver/pull/73